### PR TITLE
Added a task to pretty print JIRA issue

### PR DESCRIPTION
**Pull request by dorsha**
Includes ticket details, status, timeline, attachments and comments.

image

### CODE REVIEW

File: Scripts/script-JIRAPrintIssue.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      });
+  }
+
+  return { Type: entryTypes.note, Contents: issue, ContentsFormat: formats.json, HumanReadable: md };
+type: javascript
+tags: []
+comment: Pretty print JIRA issue into the incident war room
+enabled: true
+args:
+- name: issueId
```
Comment (asafshen) [inline Scripts/script-JIRAPrintIssue.yml]: Shouldn't it be required arg?

File: Scripts/script-JIRAPrintIssue.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      comments.map(function(comment, i) {
+          md += '\n**Comment ' + (i + 1) + '** From: ' + comment.author.displayName + ' (' + comment.author.emailAddress + ') - ' + comment.created + ':\n';
+          md += comment.body;
+          md += '\n\n --- \n\n';
+      });
+  }
+
+  return { Type: entryTypes.note, Contents: issue, ContentsFormat: formats.json, HumanReadable: md };
+type: javascript
+tags: []
```
Comment (asafshen) [inline Scripts/script-JIRAPrintIssue.yml]: we have `jira` tag. I think we should is it

File: Scripts/script-JIRAPrintIssue.yml [hunk 3]
```diff
@@ -0,0 +1,82 @@
+commonfields:
+  id: JIRAPrintIssue
+  version: -1
+name: JIRAPrintIssue
+script: |-
+  var response = executeCommand('jira-get-issue', { issueId: args.issueId });
+
+  if (!response || response.length === 0 || !response[0].Contents) {
+      return 'JIRA issue not found (' + args.issueId + ')';
```
Comment (asafshen) [inline Scripts/script-JIRAPrintIssue.yml]: I ran it with random Id and it did not catch it
instead it throws `Cannot access member 'summary' of undefined` (probably from line 18)

**Comment by dorsha** [comment]
@asafshen done.

**Pull request merged**